### PR TITLE
Merge broker-dev into maintenance branch and bring it back

### PR DESCRIPTION
**Pull request by RPangrle**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/commit/04ccf03ff4e9762916337579809c3bcfd631eb0d%23commitcomment-13749611%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391%23discussion_r42825576%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391%23discussion_r42825669%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391%23discussion_r42826417%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391%23discussion_r43070711%22%5D%2C%20%22comments%22%3A%20%7B%22Commit%2004ccf03ff4e9762916337579809c3bcfd631eb0d%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/commit/04ccf03ff4e9762916337579809c3bcfd631eb0d%23commitcomment-13749611%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%23395%20%22%2C%20%22created_at%22%3A%20%222015-10-13T19%3A12%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7004783%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/RPangrle%22%7D%7D%5D%2C%20%22title%22%3A%20%22Commit%2004ccf03%20comments%22%7D%2C%20%22Pull%20089c03b72761f68762a653678eee554f31e8d131%20ADALiOS/ADALiOS/ADALiOS.h%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391%23discussion_r42825576%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22seems%20like%20more%20than%20a%20patch%20update.%20How%20about%20moving%20to%202.1.0%3F%22%2C%20%22created_at%22%3A%20%222015-10-23T01%3A20%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5713915%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kpanwar%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%27m%20going%20to%20bump%20it%20up%20to%202.1%20when%20we%20make%20a%20public%20release%2C%20for%20now%20I%27m%20reserving%202.0.x%20for%20internal%20releases.%22%2C%20%22created_at%22%3A%20%222015-10-27T00%3A05%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7004783%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/RPangrle%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20ADALiOS/ADALiOS/ADALiOS.h%3AL20-31%22%7D%2C%20%22Pull%20089c03b72761f68762a653678eee554f31e8d131%20ADALiOS/ADALiOS/ADAuthenticationRequest%252BAcquireToken.m%2033%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391%23discussion_r42825669%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22why%20are%20broker_enabled%20ifdef%20missing%3F%22%2C%20%22created_at%22%3A%20%222015-10-23T01%3A22%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5713915%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kpanwar%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20added%20BROKER_ENABLED%20in%20a%20previous%20change.%20Those%20ifdefs%20weren%27t%20correct%20so%20I%20just%20removed%20them.%22%2C%20%22created_at%22%3A%20%222015-10-23T01%3A42%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7004783%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/RPangrle%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20ADALiOS/ADALiOS/ADAuthenticationRequest%2BAcquireToken.m%3AL220-232%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Commit 04ccf03ff4e9762916337579809c3bcfd631eb0d'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-library-for-objc/commit/04ccf03ff4e9762916337579809c3bcfd631eb0d#commitcomment-13749611'>Commit 04ccf03 comments</a></b>
- <a href='https://github.com/RPangrle'></a> #395
- [ ] <a href='#crh-comment-Pull 089c03b72761f68762a653678eee554f31e8d131 ADALiOS/ADALiOS/ADALiOS.h 4'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391#discussion_r42825576'>File: ADALiOS/ADALiOS/ADALiOS.h:L20-31</a></b>
- <a href='https://github.com/kpanwar'></a> seems like more than a patch update. How about moving to 2.1.0?
- <a href='https://github.com/RPangrle'></a> I'm going to bump it up to 2.1 when we make a public release, for now I'm reserving 2.0.x for internal releases.
- [ ] <a href='#crh-comment-Pull 089c03b72761f68762a653678eee554f31e8d131 ADALiOS/ADALiOS/ADAuthenticationRequest%2BAcquireToken.m 33'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391#discussion_r42825669'>File: ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m:L220-232</a></b>
- <a href='https://github.com/kpanwar'></a> why are broker_enabled ifdef missing?
- <a href='https://github.com/RPangrle'></a> I added BROKER_ENABLED in a previous change. Those ifdefs weren't correct so I just removed them.

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-library-for-objc/pull/391?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-library-for-objc/pull/391?approve=1'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/391'>Refresh</a>.<a href='#crh-update-none'></a>

File: ADALiOS/ADALiOS/ADALiOS.h [hunk 1]
```diff
@@ -20,11 +20,11 @@
 //version in static define until we identify a better place:
 #define ADAL_VER_HIGH       2
 #define ADAL_VER_LOW        0
-#define ADAL_VER_PATCH      0
```
Comment (kpanwar) [inline ADALiOS/ADALiOS/ADALiOS.h]: seems like more than a patch update. How about moving to 2.1.0?

File: ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m [hunk 2]
```diff
@@ -220,14 +220,12 @@ - (void)requestToken:(ADAuthenticationCallback)completionBlock
          }
          else
          {
-#if BROKER_ENABLED
```
Comment (kpanwar) [inline ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m]: why are broker_enabled ifdef missing?

Comment (RPangrle) [inline ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m]: I added BROKER_ENABLED in a previous change. Those ifdefs weren't correct so I just removed them.

File: ADALiOS/ADALiOS/ADALiOS.h (hunk 1 above)
Comment (RPangrle) [inline ADALiOS/ADALiOS/ADALiOS.h]: I'm going to bump it up to 2.1 when we make a public release, for now I'm reserving 2.0.x for internal releases.

**Pull request merged**